pages/index.tsx

```diff
@@ -61,7 +61,7 @@ const IndexPage: React.SFC<IProps> = ({
             user = candidates.find((candidat) => candidat.id === user_id);
             break;
         default:
-            return <Error statusCode={404} />;
+            break;
```

Comment:
It is like removing default case... so quite bad, you should at least throw an error